yktrace/src/hwt/mapper.rs

```diff
@@ -101,23 +101,25 @@ impl HWTMapper {
     }
 
     /// Maps each entry of a hardware trace back the IR block from whence it was compiled.
```

Comment:
...back **to** the IR block...

Also, I would prefer saying "from where" or rather even "from which" is was compiled. Whence just sounds odd.

Reply:
It sound's quite old fashioned, but does have precedent in programming.

e.g. fseek:
```
     int
     fseek(FILE *stream, long offset, int whence);
```

I'll fix it anyway :)

Reply:
I think with the renaming to yk we should continue to cut our ties to Shakespeare. ;-)